src/main/java/com/ft/methodeimagemodelmapper/service/MethodeImageModelMapper.java

```diff
@@ -34,19 +34,22 @@
 public class MethodeImageModelMapper {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MethodeImageModelMapper.class);
-    private static final String IMAGE_TYPE = "Image";
-    private static final String MEDIATYPE_PREFIX = "image/";
-    private static final String DEFAULT_MEDIATYPE = "image/jpeg";
+    public static final String IMAGE_TYPE = "Image";
```

Comment:
The fact that we made some "internal" constants public so that they can be used in the unit tests  for a different class ( `GraphicResolverTest` ) is really not ok. I think the best approach here would be to extract these common constants to a separate file